vyos/vyos-build
# T6703: add support for amd pstate driver

## Change Summary
Building amd pstate driver in the kernel. By default the acpi_cpufreq will still take precedence over the amd_pstate driver.
Kernel command line arguments are required to activate the driver.

## Types of changes

## Related Task(s)
https://vyos.dev/T6703

## Component(s) name
kernel

## Proposed changes


## How to test
Set kernel arguments `initcall_blacklist=acpi_cpufreq_init amd_pstate=passive` on an AMD machine.
Then after boot verify the pstate driver is active: `cat /sys/devices/system/cpu/cpu0/cpufreq/scaling_driver` should be `amd-pstate`

## Checklist:
